src/runtime/storageNG/tests/store-sequence-test.ts

```diff
@@ -27,14 +28,22 @@ class MockDriver<Data> extends Driver<Data> {
 }
 
 class MockStorageDriverProvider implements StorageDriverProvider {
-  willSupport(storageKey: string) {
+  willSupport(storageKey: StorageKey) {
     return true;
   }
-  driver<Data>(storageKey: string, exists: Exists): Driver<Data> {
+  driver<Data>(storageKey: StorageKey, exists: Exists): Driver<Data> {
     return new MockDriver<Data>(storageKey, exists);
   }
 }
 
+class MockStorageKey extends StorageKey {
+  constructor() {
+    super('testing');
+  }
+}
+
+const testKey = new MockStorageKey();
```

Comment:
yeah I guess I don't need this any more.  I'll remove it from here and store-test.

Reply:
Ahh, so it's because StorageKey is abstract.

It doesn't currently *need* to be abstract, but it will as I add in capabilities that are already in the old storage stack (e.g. generating child keys, etc.). So probably worth leaving this.